src/pages/home/IncidentCard.tsx

```diff
@@ -62,26 +78,42 @@ const IncidentCard = (props: Props) => {
   const incidentTimestamp =
     incident.updates[incident.updates.length - 1].timestamp
   const state = getState(incident)
+  const [expanded, setExpanded] = useState(state !== 'stable')
+  const [expandText, setExpandText] = useState(
+    expanded ? '-' : '+'
+  )
+  const changeButtonText = () =>
+    expanded ? setExpandText('-') : setExpandText('+')
   return (
     <Card state={state}>
       <TitleLink to={`/edit/${incident.id}`}>
         <h1>{incidentTimestamp.toDate().toUTCString()}</h1>
         <h2>{incident.title}</h2>
       </TitleLink>
-      <ServiceList>
-        {incident.services.map(service => (
-          <li key={service}>{service}</li>
-        ))}
-      </ServiceList>
-      <LinkButton
-        to={`/update/${incident.id}`}
-        css="display:inline-block;margin-top:1em;"
+      <ExpandButton
+        onClick={evt => {
+          setExpanded(!expanded)
+          changeButtonText()
+        }}
       >
-        Add an update
-      </LinkButton>
-      {incident.updates.map(update => (
-        <IncidentUpdate key={update.timestamp.seconds} update={update} />
-      ))}
+        {expandText}
```

Comment:
Would this work `{expanded ? '-' : '+'}}`
If it does I think it's better since we are already saving if it's expanded in the state. The '-' or '+' is related to how we display it so better to remove it from the state?

Reply:
Yes, better. I'll update it